tasks/skylab/FastqProcessing.wdl

```diff
@@ -257,7 +257,7 @@ task FastqProcessATAC {
         Int preemptible = 3
 
         # Additional parameters for fastqprocess
-        Int num_output_files
+        String num_output_files
```

Comment:
Can a user override this? If so, maybe we can keep it an "Int" to enforce that this is actually a number (specifically an integer). Is this a string for the echo statement?

Reply:
I set it as an integer. 

This parameter should depend on BWA machine type. Thus, user should not override it, but can if wanted